addons/pos_cache/models/pos_cache.py

```diff
@@ -29,7 +28,7 @@ def refresh_cache(self):
         prod_ctx = prod_ctx.sudo(self.compute_user_id.id)
         res = prod_ctx.read(self.get_product_fields())
         datas = {
-            'cache': base64.encodestring(cPickle.dumps(res)),
+            'cache': json.dumps(res),
```

Comment:
The value must be encoded in base64, because it is stored in attachment (which expects base64-encoded data).